codegen/smithy-go-codegen-test/model/main.smithy

```diff
@@ -151,6 +151,7 @@ structure GetForecastInput {
 
 structure GetForecastOutput {
     chanceOfRain: Float,
+    precipitation: Precipitation,
```

Comment:
Looks like this file is updated in a commit that's shared with #9, make sure this is clean when merging.

Reply:
Yup, it's the same commit cherry-picked onto this branch.